### ISSUE TITLE
Discussions page is not found (404 error)

### ISSUE THREAD

**Issue by marcusberger** [atom/atom.io#233]
Scenario: Discussions page
   Given the acess in atom.io
     And click menu Discussions
     Then page not found

**Comment by micnolmad** [comment]
Yeah I think the link should be https://github.com/atom/atom/issues not https://github.com/atom/atom/discussions
Maybe github have removed a feature?
